src/Illuminate/Support/composer.json

```diff
@@ -18,7 +18,8 @@
         "ext-mbstring": "*",
         "illuminate/contracts": "5.2.*",
         "doctrine/inflector": "~1.0",
-        "danielstjules/stringy": "~2.1"
+        "danielstjules/stringy": "~2.1",
+        "symfony/polyfill-php56": "^1.0"
```

Comment:
Actually,  please move this to suggest in this component, like with random_compat.